ucats/anscombe.py

```diff
@@ -1,13 +1,15 @@
 import numpy as np
 
 
-
 class Anscombe:
-    "Variance-stabilizing transformation"
+    """Variance-stabilizing transformation"""
+
     @staticmethod
     def transform(data):
-        return 2*(data+3/8)**0.5
+        return 2 * (data + 3 / 8) ** 0.5
```

Comment:
is there an option for `black` to omit spaces for exponentiation, multiplication and division?